tlcc/enclave/cgo_logging.go

```diff
@@ -0,0 +1,17 @@
+/*
+   Copyright 2019 Intel Corporation
+   Copyright IBM Corp. All Rights Reserved.
+
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package enclave
+
+// extern void golog(char*);
+//
+// int golog_cgo_wrapper(const char* str)
+// {
+//      golog((char*)str);
+//      return 1;
+// }
+import "C"
```

Comment:
This is a duplicate version.  I wonder whether could be handled via a small package in `internal` which does above and defines a single go function which gets a logger object as parameter, sets the callback plus based on a function pointer to the function defined with the logger object as a closure. Not 100% sure though whether that works and i guess as long as there are only two occurrence of this pattern it is certainly not worth spending much time on this. If we add  more places, might be worth though revisiting?

Reply:
> This is a duplicate version. 

True. It was convenient and necessary because cgo duplicates the preamble when it compiles the exported functions. 

> If we add more places, might be worth though revisiting?

Definitely.